<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LureForm</class>
 <widget class="QWidget" name="LureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>951</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="lure_list">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <column/>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="lure_new">
         <property name="toolTip">
          <string>Uusi viehe</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/document-new.png</normaloff>:/res/document-new.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lure_undo">
         <property name="toolTip">
          <string>Peruuta muutokset</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/edit-undo.png</normaloff>:/res/edit-undo.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lure_save">
         <property name="toolTip">
          <string>Tallenna viehe</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/document-save.png</normaloff>:/res/document-save.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lure_delete">
         <property name="toolTip">
          <string>Poista viehe</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/res/edit-cut.png</normaloff>:/res/edit-cut.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+X</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Vieheen tiedot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout_5">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>Valmistaja</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="lure_manufacturer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="minimumContentsLength">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_46">
          <property name="text">
           <string>Malli</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="lure_model">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="minimumContentsLength">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>Koko</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="lure_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="minimumContentsLength">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Tyyppi</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="lure_type">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="minimumContentsLength">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Lempinimi</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lure_nickname"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_48">
          <property name="text">
           <string>Väritysnimi</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="lure_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="minimumContentsLength">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>selkä-kylki-vatsa</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="lure_back">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="lure_side">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="lure_belly">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Suosikki</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ei näy listassa</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QCheckBox" name="lure_notvisible">
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QCheckBox" name="lure_favorite">
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="lure_class">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>luomu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>voimakaskontrastinen luomu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ärsytysväri</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>voimakas ärsytysväri</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="lure_contrast">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Kontrastiväri</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Väritysluokka</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lure_manufacturer</tabstop>
  <tabstop>lure_model</tabstop>
  <tabstop>lure_size</tabstop>
  <tabstop>lure_type</tabstop>
  <tabstop>lure_nickname</tabstop>
  <tabstop>lure_favorite</tabstop>
  <tabstop>lure_notvisible</tabstop>
  <tabstop>lure_new</tabstop>
  <tabstop>lure_undo</tabstop>
  <tabstop>lure_save</tabstop>
  <tabstop>lure_delete</tabstop>
  <tabstop>lure_list</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
